tests/modules/worksheets/test_worksheet.py

```diff
@@ -30,6 +34,38 @@ def worksheet(client, tenant, group):
     return response.data.createWorksheet
 
 
+@pytest.fixture(scope='module', autouse=True)
+def mock_s3_client(module_mocker):
+    s3_client = MagicMock()
+    module_mocker.patch('dataall.modules.worksheets.services.worksheet_query_result_service.S3_client', s3_client)
+
+    # s3_client.client.return_value = s3_client
+
```

Comment:
I think the correct syntax is `s3_client.get_presigned_url.return_value` without the brackets

Reply:
I tried without brackets but that didn't worked and was throwing error.
so fall backed to the approach which is working